docs/world api.md

```diff
@@ -680,3 +680,60 @@ def generate_output(self, output_directory: str):
     generate_mod(src, out_file, data)
 ```
 
+### Documentation
+
+Every World implemented should have a tutorial and a game info page. These are both rendered on the website by reading
+the `.md` files in your world's `/docs` directory.
+
+#### Game Info
+The game info page is for a short breakdown of what your game is and how it works in Archipelago. Any additional
+information that may be useful for the player to know when learning your randomizer should also go here. The file format
+is `<language key>_<game name>.md`. While you can write these docs for multiple languages and can tell the `WebWorld`
+you have multiple of these docs, they currently do not render on the website.
```

Comment:
I'm not a native speaker, but the "to know" seems redundant to me.
The other changes: i don't have the info page in my webworld?
```suggestion
information that may be useful to the player when learning your randomizer should also go here. The file name format
is `<language key>_<game name>.md`. While you can write these docs for multiple languages, currently only the english
version is displayed on the website.
```